pymbar/mbar.py

```diff
@@ -404,27 +405,25 @@ def computeEffectiveSampleNumber(self, verbose = False):
         return N_eff
 
     # =========================================================================
-    def computeOverlap(self, return_dict=True):
+    def computeOverlap(self):
         """
         Compute estimate of overlap matrix between the states.
 
         Parameters
         ----------
-        return_dict : bool, Default False
-            If true, results are a dict, else a tuple
+        None
 
         Returns
         -------
+        Results dictionary with the following keys:
```

Comment:
Double check if this syntax gets properly recognized in the rendered documentation.

Reply:
I'm not actually sure how to check the rendered documentation.  What should I run to get it?